Jenkinsfile

```diff
@@ -325,10 +325,10 @@ pipeline {
                             }
                         }
                         stage('against EE') {
-                            // TODO: Remove skip
-                            when { expression { return false } }
                             steps {
-                                echo 'Example of where we could run lite-core unit tests against a running SG EE'
+                                sh 'touch litecore.out'
+                                sh 'docker pull couchbase/sg-test-litecore:latest'
+                                sh 'docker run --net=host --rm -v /root/.ssh/id_rsa_ns-buildbot:/root/.ssh/id_rsa -v `pwd`/sync_gateway_ee-linux:/sync_gateway -v `pwd`/litecore.out:/output.out couchbase/sg-test-litecore:latest'
```

Comment:
moving this down into the `gitStatusWrapper` block will allow the exit code to be captured and reported back to GitHub as a separate build status.